src/components/DateAccordionContainer/DateAccordionsContainer.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { StyleSheet, View } from 'react-native';
+import { Accordion } from 'native-base';
+import { Ionicons } from '@expo/vector-icons'; // eslint-disable-line import/no-extraneous-dependencies
+
+import {
+  arrayOf, bool, string, shape,
+} from 'prop-types';
+import { savedRecordsByRecordDateSelector } from '../../redux/selectors';
+import BaseText from '../Generic/BaseText';
+import Colors from '../../constants/Colors';
+import SubTypeAccordionsContainer from '../SubTypeAccordion/SubTypeAccordionsContainer';
+
+const DateAccordion = ({ date, types, fromDetailsPanel }) => {
+  const dataArray = [{ title: date, content: types }];
+
+  const renderHeader = (item, expanded) => {
+    const chevronIcon = expanded
+      ? <Ionicons name="chevron-up" size={16} color={Colors.accordionChevronIcon} />
+      : <Ionicons name="chevron-down" size={16} color={Colors.accordionChevronIcon} />;
+    return (
+      <View style={styles.header}>
+        {chevronIcon}
+        <BaseText variant="title" style={styles.headerText}>
+          {item.title}
+        </BaseText>
+      </View>
+    );
+  };
+
+  const renderContent = (item) => (
+    <View style={styles.content}>
+      <SubTypeAccordionsContainer
+        data={item.content}
+        fromDetailsPanel={fromDetailsPanel}
+      />
+    </View>
+  );
```

Comment:
this might be more clear (and possibly more performant) if it is defined its own component, outside of this `render`

since it closes-over `fromDetailsPanel`, it could possible be put in to or referenced from `Accordion` (which could be prop-drilled `fromDetailsPanel`)